code/modules/atmospherics/environmental/LINDA_fire.dm

```diff
@@ -205,11 +207,11 @@
 	if((visual_update_tick++ % 7) == 0)
 		update_color()
 
-	if(temperature > location.max_fire_temperature_sustained)
+/*	if(temperature > location.max_fire_temperature_sustained)
 		location.max_fire_temperature_sustained = temperature
 
 	if(location.heat_capacity && temperature > location.heat_capacity)
-		location.to_be_destroyed = TRUE
+		location.to_be_destroyed = TRUE*/
```

Comment:
`max_fire_temperature_sustained` seems to be functionally commented out
What's the plan for it?

Reply:
ah i left it there, it will be removed since that code is managed by the Linda_turf_tile.dm